sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2134,6 +2134,37 @@ class HiveDDLSuite
     }
   }
 
+  test("the qualified path of a partition is stored in the catalog") {
```

Comment:
can we merge the 2 similar tests? We can use `isUsingHiveMetastore` to decide the USING clause.